db/migrations/0003_alter_order_user.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.0.2 on 2024-11-12 01:55
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_user_order_alter_movie_actors_alter_movie_genres_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='order',
+            name='user',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='orders', to=settings.AUTH_USER_MODEL),

```

Comment:
The `user` field in the `Order` model is now set to allow `null` and `blank` values. Ensure this change aligns with your application's logic, as it allows orders to exist without an associated user.